include/triton/core/tritonserver.h

```diff
@@ -317,20 +317,20 @@ typedef enum TRITONSERVER_errorcode_enum {
 /// \param code The error code.
 /// \param msg The error message.
 /// \return A new TRITONSERVER_Error object.
-TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_ErrorNew(
+TRITONSERVER_DECLSPEC struct TRITONSERVER_Error* TRITONSERVER_ErrorNew(
     TRITONSERVER_Error_Code code, const char* msg);
 
 /// Delete an error object.
 ///
 /// \param error The error object.
-TRITONSERVER_DECLSPEC void TRITONSERVER_ErrorDelete(TRITONSERVER_Error* error);
+TRITONSERVER_DECLSPEC void TRITONSERVER_ErrorDelete(struct TRITONSERVER_Error* error);
 
 /// Get the error code.
 ///
 /// \param error The error object.
 /// \return The error code.
-TRITONSERVER_DECLSPEC TRITONSERVER_Error_Code
-TRITONSERVER_ErrorCode(TRITONSERVER_Error* error);
+TRITONSERVER_DECLSPEC struct TRITONSERVER_Error_Code
```

Comment:
i goofed here.  shoudl shouldn't be a struct

```suggestion
TRITONSERVER_DECLSPEC TRITONSERVER_Error_Code
```